--- sqlite3.c	Fri Dec  2 11:36:36 2016
+++ sqlite3_patched.c	Fri Dec  2 17:33:18 2016
@@ -17,6 +17,8 @@
 ** language. The code for the "sqlite3" command-line shell is also in a
 ** separate file. This file contains only code for the core SQLite library.
 */
+#include "sqlite3_Export.h"
+#define SQLITE_HAS_CODEC 1
 #define SQLITE_CORE 1
 #define SQLITE_AMALGAMATION 1
 #ifndef SQLITE_PRIVATE
@@ -165719,6 +165721,7 @@ SQLITE_PRIVATE void sqlite3Fts3IcuTokenizerModule(
 #endif /* !defined(SQLITE_CORE) || defined(SQLITE_ENABLE_FTS3) */
 
 /************** End of fts3_icu.c ********************************************/
+#include "codecext.c"
 /************** Begin file sqlite3rbu.c **************************************/
 /*
 ** 2014 August 30
